klam-ssh/v2/authorizedkeys_command.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash -x
 
+USER=$1
+SYSDFILE="/etc/systemd/system/docker.service.d/10-enable-namespaces.conf"
+
+# docker start functions
+start_nsdocker ()
+{
+  docker run --privileged --userns=host --rm -e ROLE_NAME=${ROLE_NAME} -e ENCRYPTION_ID=${ENCRYPTION_ID} -e ENCRYPTION_KEY=${ENCRYPTION_KEY} -e KEY_LOCATION_PREFIX=${KEY_LOCATION_PREFIX} ${IMAGE} /usr/lib/klam/getKeys.py ${USER}
```

Comment:
i can remove it, i was basing on what was in ethos-systemd https://git.corp.adobe.com/adobe-platform/ethos-systemd/search?utf8=%E2%9C%93&q=--userns.  I did run those privileged and userns docker comamnds on my environment and it seemed to work as well.